cabal.sandbox.config

```diff
@@ -0,0 +1,25 @@
+-- This is a Cabal package environment file.
+-- THIS FILE IS AUTO-GENERATED. DO NOT EDIT DIRECTLY.
+-- Please create a 'cabal.config' file in the same directory
+-- if you want to change the default settings for this sandbox.
```

Comment:
:x I'll update the .gitignore

On Sat, Jun 6, 2015 at 11:18 AM, Gary Burgess <notifications@github.com>
wrote:

> In cabal.sandbox.config
> <https://github.com/paf31/monad-unify/pull/2#discussion_r31869800>:
>
> > @@ -0,0 +1,25 @@
> > +-- This is a Cabal package environment file.
> > +-- THIS FILE IS AUTO-GENERATED. DO NOT EDIT DIRECTLY.
> > +-- Please create a 'cabal.config' file in the same directory
> > +-- if you want to change the default settings for this sandbox.
>
> You probably didn't mean to commit this [image: :wink:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paf31/monad-unify/pull/2/files#r31869800>.
>